testutil/operator_builder.go

```diff
@@ -63,6 +63,20 @@ func (_m *OperatorBuilder) ID() string {
 	return r0
 }
 
+// SetID provides a mock function with given fields: _a0
+func (_m *OperatorBuilder) SetID(_a0 string) error {
+	ret := _m.Called(_a0)
```

Comment:
We're getting dinged on code coverage, but this is just test helper code. Let's rename this file to end with `_test.go` so it is excluded from coverage.

Reply:
renamed this file